core-services/03-filesystems.sh

```diff
@@ -74,7 +74,7 @@ if [ -z "$FASTBOOT" ]; then
 fi
 
 msg "Mounting rootfs read-write..."
-mount -o remount,rw / || emergency_shell
+LIBMOUNT_FORCE_MOUNT2=always mount -o remount,rw / || emergency_shell
 
 msg "Mounting all non-network filesystems..."
-mount -a -t "nosysfs,nonfs,nonfs4,nosmbfs,nocifs" -O no_netdev || emergency_shell
+LIBMOUNT_FORCE_MOUNT2=always mount -a -t "nosysfs,nonfs,nonfs4,nosmbfs,nocifs" -O no_netdev || emergency_shell
```

Comment:
Why is the change needed here? Nothing should be remounted.

Reply:
yeah, realised that was unnecessary